README.md

```diff
@@ -85,932 +119,667 @@ Unlike many authorization systems where a service controls access to resources i
-
-Abilities MUST NOT be case sensitive. For example, `http/post`, `http/POST`, `HTTP/post`, `HTTP/POST`, and `hTtP/pOsT` MUST all mean the same ability.
-
-There MUST be at least one path segment as a namespace. For example, `http/put` and `db/put` MUST be treated as unique from each other.
-
-The only reserved ability MUST be the un-namespaced [`"*"` (top)][top ability], which MUST be allowed on any resource.
-
-## 2.4 Caveats
-
-Capabilities MAY define additional optional or required fields specific to their use case in the caveat fields. This field is OPTIONAL in the general case, but MAY be REQUIRED by particular capability types that require this information to validate. Caveats MAY function as an "escape hatch" for when a use case is not fully captured by the resource and ability fields. Caveats can be read as "on the condition that `<some caveat>` holds".
-
-## 2.5 Capability
-
-A capability is the association of an "ability" to a "resource": `resource x ability x caveats`.
-
-The resource and ability fields are REQUIRED. Any non-normative extensions are OPTIONAL.
-
-```
-{ $RESOURCE: { $ABILITY: [ $CAVEATS ] } }
-```
-
-### 2.5.1 Examples
-
-``` json
-{
-  "example://example.com/public/photos/": {
-    "crud/read": [{}],
-    "crud/delete": [
-      {
-        "matching": "/(?i)(\\W|^)(baloney|darn|drat|fooey|gosh\\sdarnit|heck)(\\W|$)/"
-      }
-    ]
-  },
-  "example://example.com/private/84MZ7aqwKn7sNiMGsSbaxsEa6EPnQLoKYbXByxNBrCEr": {
-    "wnfs/append": [{}]
-  },
-  "mailto:username@example.com": {
-    "msg/send": [{}],
-    "msg/receive": [
-      {
-        "max_count": 5,
-        "templates": [
-          "newsletter",
-          "marketing"
-        ]
-      }
-    ]
-  }
-}
-```
-
-## 2.6 Authority
+A resource describes the noun of a capability. The resource pointer MUST be provided in [URI] format. Arbitrary and custom URIs MAY be used, provided that the intended recipient can decode the URI. The URI is merely a unique identifier to describe the pointer to — and within — a resource.
 
-The set of capabilities delegated by a UCAN is called its "authority." This functions as a declarative description of delegated abilities.
+Having a unique agent represent a resource (and act as its manager) is RECOMMENDED. However, to help traditional ACL-based systems transition to certificate capabilities, an agent MAY manage multiple resources, and [act as the registrant in the ACL system][wrapping existing systems].
 
-Merging capability authorities MUST follow set semantics, where the result includes all capabilities from the input authorities. Since broader capabilities automatically include narrower ones, this process is always additive. Capability authorities can be combined in any order, with the result always being at least as broad as each of the original authorities.
+Unless explicitly stated, the Resource of a UCAN MUST be the Subject.
```

Comment:
I don't understand where and how any other state of affairs could be explicitly stated? Do you mean 

```suggestion
Unless all expected recipients of a UCAN will understand how the Subject of a UCAN relates to the Resource being authorized because this relationship is explicitly clarified out-of-band, it can be assumed by all parties that a UCAN's subject is coterminous with the Resource authorized by that UCAN.
```
?  I assume not, but if you asked _me_ to make that sentence more verbose, this is what I would come up with, which feels like a very wrong guess.